docker-compose.yml

```diff
@@ -66,6 +66,7 @@ services:
     environment:
       - DSHOST=store
       - DSPORT=50050
+      - FORWARDED_ALLOW_IPS=${PROXY_IP:-127.0.0.1}
```

Comment:
Hmm, still one more: This should not refer to `PROXY_IP` anymore? To be consistent with README?